src/apps/_experiments/OswAppWeather.cpp

```diff
@@ -263,21 +279,22 @@ void OswAppWeather::weatherRequest() {
 
 bool OswAppWeather::_request() {
     HTTPClient http;
-    OSW_LOG_I(this->url);
-    http.begin(this->url, OPENWEATHERMAP_CA_CERT);
     int code = 0;
+    OSW_LOG_D("Request: ");
+    OSW_LOG_D(this->url);
+    http.begin(this->url, OPENWEATHERMAP_CA_CERT);
     if (OswServiceAllTasks::wifi.isConnected()) {
         OswHal::getInstance()->disableDisplayBuffer();
         this->forecast.clear();
         this->dataLoaded=false;
-        OSW_LOG_I("free heap ", ESP.getFreeHeap());
+        OSW_LOG_D("free heap ", ESP.getFreeHeap());
         code = http.GET();
     } else {
         return false;
     }
     http.end();
     OswServiceAllTasks::wifi.disconnectWiFi();
-    OSW_LOG_I("code:", code);
+    OSW_LOG_D("Request returned code: ", code);
     if (code == 200) {
```

Comment:
Can we use replace word? 

```
SUCCESS = 200
{ . . . }
if (code == SUCCESS)
```

Reply:
Hmm, maybe the HTTP lib already has defines for that, lemme check

Reply:
Bingo: `HTTP_CODE_OK`